pydriller/repository.py

```diff
@@ -167,7 +170,8 @@ def _clone_folder(self) -> str:
     def _prep_repo(self, path_repo: str) -> Generator[Git, None, None]:
         local_path_repo = path_repo
         if self._is_remote(path_repo):
-            local_path_repo = self._clone_remote_repo(self._clone_folder(), path_repo)
+            clone_folder = self._clone_folder()
+            local_path_repo = self._clone_remote_repo(clone_folder, path_repo)
```

Comment:
Can you undo this change? Thanks!
```suggestion
            local_path_repo = self._clone_remote_repo(self._clone_folder(), path_repo)
```